code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -1,4 +1,4 @@
-/obj/item/melee/energy
+/obj/item/energy
```

Comment:
`/obj/item/melee/energy` gives you a fair grasp of what you're talking about, `/obj/item/energy` does not

Perhaps `/obj/item/eblade` or something

Reply:
`/obj/item/lightsaber`